src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1109,6 +1159,8 @@ private static String addPrefix(String server) {
      * See http://msdn.microsoft.com/en-us/library/aa746475(VS.85).aspx for the syntax by example.
      * WANTED: The specification of the syntax.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Filter may be changed in the runtime by scripts if required")
```

Comment:
I do not agree with the current `@SuppressFBWarnings`. Either fix the justification and add the TODO to make the field final or just remove the `@SuppressFBWarnings` and make the field final